model/note/note.go

```diff
@@ -362,6 +364,56 @@ func writeFile(inst *instance.Instance, doc *Document, oldDoc *vfs.FileDoc) (fil
 	return
 }
 
+// List returns a list of notes sorted by descending updated_at. It uses
+// pagination via a mango bookmark.
+func List(inst *instance.Instance, bookmark string) ([]*vfs.FileDoc, string, error) {
+	lock := inst.NotesLock()
+	if err := lock.Lock(); err != nil {
+		return nil, "", err
+	}
+	defer lock.Unlock()
+
+	var docs []*vfs.FileDoc
+	req := &couchdb.FindRequest{
+		UseIndex: "by-mime-updated-at",
+		Selector: mango.And(
+			mango.Equal("mime", noteMime),
```

Comment:
Does that mean we're now storing a note as an io.cozy.files with mime = text/vnd.cozy.note+markdown ?

Reply:
Yes, the stack will use this new mime-type for notes.